app/template/[templateId]/access/page.tsx

```diff
@@ -0,0 +1,174 @@
+'use client';
+
+import React from 'react';
+import {
+  Breadcrumb,
+  Breadcrumbs,
+  Button,
+  Form,
+  Input,
+  Label,
+  Link,
+  TextField,
+} from 'react-aria-components';
+import PageHeader from "@/components/PageHeader";
+import styles from './TemplateAccessPage.module.scss';
+
+interface ExternalPerson {
+  id: string;
+  fullName: string;
+  email: string;
+}
+
+const externalPeople: ExternalPerson[] = [
+  {
+    id: '1',
+    fullName: 'Frederick Cook',
+    email: 'Fred.Cook@cam.ac.uk'
+  },
+  {
+    id: '2',
+    fullName: 'Robert Peary',
+    email: 'peary.robert@psu.edu'
+  }
+];
+
+interface Template {
+  id: string;
+  name: string;
+  author: string;
+  version: string;
+  publishedDate: string;
+
+}
+
+const template: Template = {
+  id: 'tpl_abcdef123456',
+  name: 'Arctic Data Center: NSF Polar Programs',
+  author: 'National Science Foundation (nsf.gov)',
+  version: '6.2',
+  publishedDate: 'Jan 1, 2024',
+};
+
+const TemplateAccessPage: React.FC = () => {
+  const handleRevokeAccess = (id: string) => {
+    console.log('Revoking access for:', id);
+    alert(`Revoking access for: ${id}`);
+    // Implementation for removing access
+  };
+
+  const handleSubmit = (e: React.FormEvent) => {
+    e.preventDefault();
+    // Handle invitation logic
+  };
+
+
+  return (
+    <div>
+      <PageHeader
+        title={"Manage Access"}
+        description=""
+        showBackButton={true}
+        breadcrumbs={
+          <Breadcrumbs>
```

Comment:
@fraserclark , what do you think about the way the breadcrumb looks on a mobile device?

Just for fun, I added some more items to the breadcrumb, and noticed it starts to get really squished, with the arrow not aligning vertically, but being located at the bottom of the wrapping text.

![image](https://github.com/user-attachments/assets/934cd206-0c46-4393-a3d8-2560562fb930)